stories/Pager.stories.mdx

```diff
@@ -4,14 +4,15 @@ import { html } from 'lit-html';
 <Meta title="Components/Pager" component="Pager" />
 
 export const DefaultPagerTemplate = (args) => html`
-  <ul class="jazz-pager" role="navigation" aria-label="pagination">
+<nav class="jazz-pager" role="navigation" aria-label="Pagination Navigation, Current Page 2">
+  <ul>
        <li>
-          <a>Previous</a>
+          <a aria-label="Previous" href="javascript:void(0)"><i class="jazz-icon jazz-icon-caret-left"></i>Previous</a>
```

Comment:
The aria-label should not be necessary here because the content is already "Previous".  The label would only be necessary if we removed the text.